Instruments/source/TupleSkimmer.cxx

```diff
@@ -437,16 +449,19 @@ class TupleSkimmer {
             event.storageMode = static_cast<UInt_t>(storage_mode.Mode());
         }
 
+
+        if (!ntuple::MetFilters(event.metFilters).PassAll()) return false;
+
         if(!setup.energy_scales.count(es) || full_event.extraelec_veto || full_event.extramuon_veto) return false;
 
-        if(setup.apply_bb_cut && (full_event.jets_p4.size() < 2
-                || std::abs(full_event.jets_p4.at(0).eta()) >= cuts::btag_2016::eta
-                || std::abs(full_event.jets_p4.at(1).eta()) >= cuts::btag_2016::eta)) return false;
+        if(setup.apply_bb_cut && (!eventInfo->HasBjetPair()
+                || std::abs(eventInfo->GetHiggsBB().GetFirstDaughter()->p4().eta()) >= cuts::btag_2016::eta
+                || std::abs(eventInfo->GetHiggsBB().GetSecondDaughter()->p4().eta()) >= cuts::btag_2016::eta)) return false;
```

Comment:
don't need to have an explicit cut on eta. It is already done in event info constructor to select bjet pair.